db/migrate/20190814202628_create_events.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class CreateEvents < ActiveRecord::Migration[5.2]
+  def change
+    create_table :events do |t|
+      t.string :title
+      t.text :description
+      t.datetime :date
+      t.integer :creator_id
```

Comment:
Its good to add indexes to foreign keys because they are used extensively to locate a record in this table. Read more about it [here](https://semaphoreci.com/blog/2017/05/09/faster-rails-is-your-database-properly-indexed.html). Do some research on how to add them in a migration. [REQUIRED]